Makefile

```diff
@@ -1,21 +1,25 @@
 
 PACKAGE		:= abuild
 VERSION		:= 3.4.0_rc4
+P=$(PACKAGE)-$(VERSION)
+
+ABUILD_DEPS	:=
 
 prefix		?= /usr
 bindir		?= $(prefix)/bin
 sysconfdir	?= /etc
 datadir		?= $(prefix)/share/$(PACKAGE)
 mandir		?= $(prefix)/share/man
 
-SCRIPTS		:= abuild abuild-keygen abuild-sign newapkbuild \
-		   abump apkgrel buildlab apkbuild-cpan checkapk \
-		   apkbuild-gem-resolver
-USR_BIN_FILES	:= $(SCRIPTS) abuild-tar abuild-gzsplit abuild-sudo abuild-fetch abuild-rmtemp
+BINS		:= $(addprefix abuild-,fetch gzsplit rmtemp sudo tar)
+SCRIPTS		:= abuild $(addprefix abuild-,keygen sign) abump \
+		   $(addprefix apkbuild-,cpan gem-resolver pypi) \
```

Comment:
using `$(addprefix abuild-,...)` here provides no value and makes it less readable. Please remove those.